fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/ClientLoanIntegrationTest.java

```diff
@@ -4843,7 +4843,7 @@ public void testLoanRefundByTransferCashBasedAccounting() {
         // refund 20 means paid: principal 1980, interest 240, fees 50, penalty
         // 0
 
-        Float TRANSFER_AMOUNT = 20f;
+        Float transferAmountValue = 20f;
```

Comment:
If this is a real constant, make it final and upper case?

Reply:
No it is not, it is changed a few lines later 
https://github.com/apache/fineract/blob/acd9de407f29cf1ba90f682b13fc03c55916f4e2/fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/ClientLoanIntegrationTest.java#L4887

Reply:
Ah ok fine, then ignore this (I'm doing some late night review on my mobile phone)